mwclient/client.py

```diff
@@ -1001,16 +1092,26 @@ def upload(self, file=None, filename=None, description='', ignore=False,
             file.close()
         return response
 
-    def chunk_upload(self, file, filename, ignorewarnings, comment, text):
+    def chunk_upload(
+        self,
+        file: io.BufferedReader,
```

Comment:
why are we using `io.BufferedReader` here but `Union[BinaryIO, TextIO]` for essentially the same thing in `upload()`? It seems like maybe you got kinda caught between two options and it's inconsistent? I don't know which is better, but it seems like we should consistently use one or the other when we mean "the thing you get from opening a file"?